packages/contract/README.md

```diff
@@ -308,7 +308,7 @@ instance.getValue.call().then(function(val) {
 });
 ```
 
-Even more helpful, however is we *don't even need* to use `.call` when a function is marked as `view` or `pure`, because `@truffle/contract` will automatically know that that function can only be interacted with via a call:
+Even more helpful, however we *don't even need* to use `.call` when a function is marked as `view` or `pure`, because `@truffle/contract` will automatically know that that function can only be interacted with via a call:
```

Comment:
I think we want to leave in the "is" in this case as this part of the sentence requires a verb to be grammatically correct and to say exactly *what* is more helpful.

Reply:
The `is` isn't necessarily here; the verb phrase is that we don't even need etc.  But there should be a comma after the however.

Reply:
If you write "Even more helpful, ..." it seems that you are referencing something that was just mentioned. That is not the case here. This text means to say, "hey, here is something else that is helpful".

Reply:
? This is referencing the `.call` functionality that was described immediately preceding.

Reply:
Yeah, it is referencing `.call` which is goes on to describe afterwards.